<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1254</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3,2,3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_1">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_11">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>串口配置</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinimumSize</enum>
             </property>
             <item>
              <widget class="QLabel" name="label_1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>串口号:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinimumSize</enum>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>波特率:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_2"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>校验位:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_3"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>数据位:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_4"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>结束位:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_5"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_refresh">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>刷新端口</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="1,0">
             <item>
              <widget class="QPushButton" name="pushButton_16b">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>关闭串口</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_16a">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>打开串口</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_12">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>信息接接收</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QListView" name="listView_12">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_21">
          <property name="title">
           <string>节点1处温度</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="1,3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_211">
             <item>
              <widget class="QPushButton" name="pushButton_21a">
               <property name="text">
                <string>查看历史</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_21b">
               <property name="text">
                <string>导出曲线</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_21"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_22">
          <property name="title">
           <string>节点2处温度</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="1,3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_221">
             <item>
              <widget class="QPushButton" name="pushButton_22a">
               <property name="text">
                <string>查看历史</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_22b">
               <property name="text">
                <string>导出曲线</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_22"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_23">
          <property name="title">
           <string>节点3处温度</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="1,3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_231">
             <item>
              <widget class="QPushButton" name="pushButton_23a">
               <property name="text">
                <string>查看历史</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_23b">
               <property name="text">
                <string>导出曲线</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_23"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_24">
          <property name="title">
           <string>节点4处温度</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_24" stretch="1,3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_241">
             <item>
              <widget class="QPushButton" name="pushButton_24a">
               <property name="text">
                <string>查看历史</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_24b">
               <property name="text">
                <string>导出曲线</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_24"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,2,4">
        <item>
         <widget class="QLabel" name="label_31">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>当前温度最高值</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="thermometreDlg" name="widget_31" native="true"/>
        </item>
        <item>
         <widget class="QFrame" name="frame_31">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_41">
          <property name="title">
           <string>节点5处温度</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_41" stretch="1,3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_411">
             <item>
              <widget class="QPushButton" name="pushButton_41a">
               <property name="text">
                <string>查看历史</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_41b">
               <property name="text">
                <string>导出曲线</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_41"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_42">
          <property name="title">
           <string>节点6处温度</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_42" stretch="1,3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_421">
             <item>
              <widget class="QPushButton" name="pushButton_42a">
               <property name="text">
                <string>查看历史</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_42b">
               <property name="text">
                <string>导出曲线</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_42"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_43">
          <property name="title">
           <string>节点7处温度</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_43" stretch="1,3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_431">
             <item>
              <widget class="QPushButton" name="pushButton_43a">
               <property name="text">
                <string>查看历史</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_43b">
               <property name="text">
                <string>导出曲线</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_43"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_44">
          <property name="title">
           <string>节点8处温度</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_44" stretch="1,3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_441">
             <item>
              <widget class="QPushButton" name="pushButton_44a">
               <property name="text">
                <string>查看历史</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_44b">
               <property name="text">
                <string>导出曲线</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_44"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1254</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_once"/>
   <addaction name="action_continuous"/>
   <addaction name="action_suspend"/>
   <addaction name="action_clear_list"/>
  </widget>
  <action name="action_once">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ft_icon/img_pkg/200.jpeg</normaloff>:/ft_icon/img_pkg/200.jpeg</iconset>
   </property>
   <property name="text">
    <string>采集一次</string>
   </property>
   <property name="toolTip">
    <string>开始采集温度</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_clear_list">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ft_icon/img_pkg/300.jpg</normaloff>:/ft_icon/img_pkg/300.jpg</iconset>
   </property>
   <property name="text">
    <string>清空消息栏</string>
   </property>
  </action>
  <action name="action_continuous">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ft_icon/img_pkg/400.jpeg</normaloff>:/ft_icon/img_pkg/400.jpeg</iconset>
   </property>
   <property name="text">
    <string>连续采集</string>
   </property>
   <property name="toolTip">
    <string>连续采集温度</string>
   </property>
  </action>
  <action name="action_suspend">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/ft_icon/img_pkg/600.jpeg</normaloff>:/ft_icon/img_pkg/600.jpeg</iconset>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
   <property name="toolTip">
    <string>暂停连续采集</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>thermometreDlg</class>
   <extends>QWidget</extends>
   <header location="global">thermometreDlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
